maintainers/maintainer-list.nix

```diff
@@ -16977,6 +16977,12 @@
     githubId = 8017899;
     name = "Sivaram Balakrishnan";
   };
+  sixstring982 = {
+    email = "sixstring982@gmail.com";
+    github = "sixstring982";
+    githubId = 1328977;
+    name = "Trent Small";
```

Comment:
please move this into it's own commit called `maintainers: add sixstring982

Reply:
Sounds good -- Done!

Reply:
On second thought, I'll do this in a separate PR.